swing/src/main/java/info/openrocket/swing/gui/simulation/SimulationOptionsPanel.java

```diff
@@ -116,10 +117,26 @@ class SimulationOptionsPanel extends JPanel {
 		label = new JLabel(trans.get("simedtdlg.lbl.Simmethod"));
 		label.setToolTipText(tip);
 		subsub.add(label, "gapright para");
-		
-		label = new JLabel("6-DOF Runge-Kutta 4");
+
+
+		EnumModel<RKStepperChoice> rkchoice = new EnumModel<>(
```

Comment:
Rename variable to be more generic

Reply:
Should be all set in `f4d47579d68af4c9350bacf9f3e73703ea254a91`.